release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1625,6 +1641,10 @@ With this release, instead of returning an overflow `servfail` error, the CoreDN
 
 * Previously, pods assigned an IP from the pool created by the Whereabouts CNI plugin persisted in the `ContainerCreating` state after a node force reboot. With this release, the Whereabouts CNI plugin issue associated with the IP allocation after a node force reboot is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-18893[*OCPBUGS-18893*])
 
+* Previously, when using the assisted installer, OVN-Kubernetes took a long time to bootstrap. This issue occurred because there were three `ovnkube-control-plane` nodes. The first two started up normally, however the third delayed delayed the installation time. The issue would only resolve after a timeout expiration; afterwards, installation would continue.
```

Comment:
```suggestion
* Previously, when using the assisted installer, OVN-Kubernetes took a long time to bootstrap. This issue occurred because there were three `ovnkube-control-plane` nodes. The first two started up normally, but the third delayed the installation time. The issue would only resolve after a timeout expiration; afterwards, installation would continue.
```